### PR TITLE
fix: Select dropdown styling

### LINKED ISSUE #4352: [v2 component] Select Dropdown - Select state should not be black
Current

Image

Should be

Image

[View component in Figma](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=385%3A33491)

### PR DESCRIPTION
**Pull request by john-afolabi**
## What does this PR do?

Fixes #4352

**Environment**: Staging(main branch)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

image

### CODE REVIEW

File: apps/web/components/ui/form/Select.tsx [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import React, { useCallback, useEffect, useState } from "react";
-import ReactSelect, { components, GroupBase, Props, InputProps, SingleValue, MultiValue } from "react-select";
+import { Check } from "react-feather";
```
Comment (PeerRich) [inline apps/web/components/ui/form/Select.tsx:2]: we have our own import for Icons. Look for

`import { Icon } from "@calcom/ui/Icon";`

and then 

`<Icon.FiCheck className="h-5 w-5" />`

**Review by PeerRich — DISMISSED** [review]

File: apps/web/components/ui/form/Select.tsx (hunk 1 above)
Comment (john-afolabi) [reply apps/web/components/ui/form/Select.tsx]: Done

Comment (PeerRich) [reply apps/web/components/ui/form/Select.tsx]: I still see direct import from react-feather, did you forget to commit?

**Review by sean-brydon — CHANGES_REQUESTED** [review]
I believe these changes are meant to be applied to the V2 component @Jaibles  

If so we should be clearer in issues in the future

**Comment by john-afolabi** [comment]
I reverted the previous changes and updated the v2 Select component instead.

**Comment by Jaibles** [comment]
@sean-brydon yes these changes are supposed to be on the component. 

Let me know what I should include to make it clearer for future issues. I thought putting "select dropdown component" at the beginning of the issue title and providing an image of the component would make it clear so an example would be appreciated  so I can avoid confusion going forward 🙏

File: packages/ui/v2/core/form/Select.tsx [hunk 2]
```diff
@@ -43,7 +63,7 @@ function Select<
         borderRadius: 6,
         colors: {
           ...theme.colors,
-          primary: "var(--brand-color)",
+          primary: "rgba(244, 245, 246, var(--tw-bg-opacity))",
```
Comment (alishaz-polymath) [inline packages/ui/v2/core/form/Select.tsx:66]:
```suggestion
          primary: "rgba(229, 231, 235, var(--tw-bg-opacity))",
```

Comment (alishaz-polymath) [reply packages/ui/v2/core/form/Select.tsx:66]: Just a minor NIT, amazing work @john-afolabi 
Thanks for your contribution 🙏

**Review by alishaz-polymath — CHANGES_REQUESTED** [review]
Minor NIT

**Review by alishaz-polymath — APPROVED** [review]
Thanks @john-afolabi for your kind contribution.
It looks good to me now 🙏
@sean-brydon @Jaibles  Can you please confirm if this looks good

[Loom](https://www.loom.com/share/a56f30cf51a14e4fa4aaba316eb29935)

**Review by sean-brydon — APPROVED** [review]

**Comment by Jaibles** [comment]
I wouldn't have black on downpress, I'd just use the select colour for that but otherwise this looks perfect and the downpress is a minor thing.

**Comment by alishaz-polymath** [comment]
@Jaibles will raise a quick fix PR for this in a bit🙏

**Pull request merged**